### DISCUSSION TITLE
Creating Simpoints for ChampSim traces

### DISCUSSION THREAD

**Discussion by vincentabraham** [ChampSim/ChampSim#391] [Q&A] [answered]
Could anyobody help me with how I could use Simpoints with ChampSim to create checkpoints for my simulation traces to speed up my simulations?

**Comment by ngober** [comment, marked as answer]
I think you are attempting to apply a methodology to the simulations that doesn't match. Simpoints are a tracing methodology. The thesis of the work is that certain small portions of the overall program can be taken as representative of the overall program. To use Simpoints with ChampSim, you must perform the Simpoint analysis prior to tracing the program, then begin the trace at that location (for example, using the `-s` flag in the provided PIN tracer). I know that the traces at https://dpc3.compas.cs.stonybrook.edu/champsim-traces/speccpu/ use this methodology. The starting instruction number is given in the file name.
